nms/jest.config.js

```diff
@@ -32,6 +32,8 @@ module.exports = {
       transform: {
         '^.+\\.js$': 'babel-jest',
       },
+      resetMocks: true,
+      restoreMocks: true,
```

Comment:
Imho `resetMocks: true,` is redundant as it is included in `restoreMocks: true,` if I read [the documentation](https://jestjs.io/docs/mock-function-api#mockfnmockreset) correctly.